coderd/notifications/testdata/rendered-templates/smtp/TemplateWorkspaceDeleted_CustomAppearance.html.golden

```diff
@@ -0,0 +1,90 @@
+From: system@coder.com
+To: bobby@coder.com
+Subject: Workspace "bobby-workspace" deleted
+Message-Id: 02ee4935-73be-4fa1-a290-ff9999026b13@blush-whale-48
+Date: Fri, 11 Oct 2024 09:03:06 +0000
+Content-Type: multipart/alternative;  boundary=bbe61b741255b6098bb6b3c1f41b885773df633cb18d2a3002b68e4bc9c4
+MIME-Version: 1.0
+
+--bbe61b741255b6098bb6b3c1f41b885773df633cb18d2a3002b68e4bc9c4
+Content-Transfer-Encoding: quoted-printable
+Content-Type: text/plain; charset=UTF-8
+
+Hi Bobby,
+
+Your workspace bobby-workspace was deleted.
+
+The specified reason was "autodeleted due to dormancy (autobuild)".
+
+
+View workspaces: http://test.com/workspaces
+
+View templates: http://test.com/templates
+
+--bbe61b741255b6098bb6b3c1f41b885773df633cb18d2a3002b68e4bc9c4
+Content-Transfer-Encoding: quoted-printable
+Content-Type: text/html; charset=UTF-8
+
+<!doctype html>
+<html lang=3D"en">
+  <head>
+    <meta charset=3D"UTF-8" />
+    <meta name=3D"viewport" content=3D"width=3Ddevice-width, initial-scale=
+=3D1.0" />
+    <title>Workspace "bobby-workspace" deleted</title>
+  </head>
+  <body style=3D"margin: 0; padding: 0; font-family: -apple-system, system-=
+ui, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarel=
+l', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif; color: #020617=
+; background: #f8fafc;">
+    <div style=3D"max-width: 600px; margin: 20px auto; padding: 60px; borde=
+r: 1px solid #e2e8f0; border-radius: 8px; background-color: #fff; text-alig=
+n: left; font-size: 14px; line-height: 1.5;">
+      <div style=3D"text-align: center;">
+        <img src=3D"https://custom.application/logo.png" alt=3D"Custom Appl=
+ication Logo" style=3D"height: 40px;" />
+      </div>
+      <h1 style=3D"text-align: center; font-size: 24px; font-weight: 400; m=
+argin: 8px 0 32px; line-height: 1.5;">
+        Workspace "bobby-workspace" deleted
+      </h1>
+      <div style=3D"line-height: 1.5;">
+        <p>Hi Bobby,</p>
+
+<p>Your workspace <strong>bobby-workspace</strong> was deleted.</p>
+
+<p>The specified reason was &ldquo;<strong>autodeleted due to dormancy (aut=
+obuild)</strong>&rdquo;.</p>
+      </div>
+      <div style=3D"text-align: center; margin-top: 32px;">

```

Comment:
> style=3D

This smells like bad URL encoding to me?

Reply:
I think it's ok:
https://en.wikipedia.org/wiki/Quoted-printable#Quoted-printable_encoding

Reply:
<obligatory link to XKCD#927>